packages/fuchsia_remote_debug_protocol/lib/src/fuchsia_remote_connection.dart

```diff
@@ -258,8 +291,6 @@ class FuchsiaRemoteConnection {
       final DartVm vmService = await _getDartVm(fp.port);
       isolates.add(vmService.getMainIsolatesByPattern(pattern));
     }
-    final Completer<List<IsolateRef>> completer =
-        new Completer<List<IsolateRef>>();
     final List<IsolateRef> result =
         await Future.wait(isolates).then((List<List<IsolateRef>> listOfLists) {
```

Comment:
Do you want to add the timeout here as well?

```dart
Future.wait(isolates).timeout(timeout).then(...)
```

Reply:
Yeah, I was thinking about that.  Sounds good.